packages/fuel-gauge/src/transaction.test.ts

```diff
@@ -0,0 +1,24 @@
+import { sleep, Provider, TransactionType } from 'fuels';
+import { launchNode } from 'fuels/test-utils';
+
+/**
+ * @group node
+ */
+describe('Transaction', () => {
+  it('should ensure a mint transaction can be decoded just fine', async () => {
+    const { cleanup, ip, port } = await launchNode({
+      args: ['--poa-instant', 'false', '--poa-interval-period', '100ms'],
+      loggingEnabled: false,
+    });
+
+    await sleep(500);
+    const nodeProvider = await Provider.create(`http://${ip}:${port}/v1/graphql`);
+    const {
+      transactions: [tx],
+    } = await nodeProvider.getTransactions({ first: 1 });
```

Comment:
@petertonysmith94 Yes because no transactions were submitted to the node. 

I've set the [poa interval period](https://github.com/FuelLabs/fuels-ts/blob/0e944b56e2ff507d4927fe8dcaf8d884886c821b/packages/fuel-gauge/src/transaction.test.ts#L10) to 100ms, which means a new block will be generated each 100ms, and with it, a Mint Transaction.